spec/models/atmosphere/virtual_machine_template_spec.rb

```diff
@@ -50,6 +50,16 @@
     end
   end
 
+  context 'tenant assignment validation' do
+    it 'does not permit a vmt to be saved with blank tenant assignment' do
+      vmt = build(:virtual_machine_template)
+      expect(vmt.valid?).to be_truthy
+      vmt.tenants = []
+      expect(vmt.valid?).to be_falsy
+      expect(vmt.errors[:tenants]).to include 'A Virtual Machine Template must be attached to at least one Tenant'
```

Comment:
Why valid true is checked in this spec? Is it not enough to change it into:

```ruby
vmt = build(:virtual_machine_template, tenants: [])
expect(vmt).to_not be_valid
expect(vmt.errors[:tenants]).to include 'A Virtual Machine Template must be attached to at least one Tenant'
```

?